sale_timesheet_rounded/models/project_project.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+
+    _inherit = 'project.project'
+
+    timesheet_rounding_unit = fields.Float(
+        string='Timesheet rounding unit',
+        default=0.0,
+        help="""1.0 = hour
+            0.25 = 15 min
+            0.084 ~= 5 min
+            0.017 ~= 1 min
+            """
+    )
+    timesheet_rounding_method = fields.Selection(
+        string='Timesheet rounding method',
+        selection=[
```

Comment:
should we forbid to change rounding if aal exist for this project, can changing of policy accidentally affected so_lines?

Reply:
Good point :+1: 
Added a help in 163317d